SetDefaultSinkId/explainer.md

```diff
@@ -0,0 +1,184 @@
+Current version: this document
+
+## Introduction
+
+Setting Audio Output Device In Frames
+
+In embedded web experiences, the top-level frame grants iframes access to system media devices like microphones, cameras and speakers. This is typically done through the permission policy (https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy) in the iframe tag. However, this access comes with two key challenges:
+
+- Independent Choice: Each iframe independently chooses its own media device. The top-level frame cannot directly influence or view this selection due to browser security restrictions (cross-origin boundary).
+- Unsynchronized Changes: When the top-level frame changes its media device, the iframes remain unaware unless they communicate using methods like postMessage. This lack of automatic synchronization can lead to inconsistencies and a disjointed user experience.
+
+We’d like to propose a new API to set the default audio output device for the current top frame and of its sub frames.
+
+## Goals
+
+- Developers can modify the default audio output for the top-level frame page.
+- Developers can modify the default audio output for the sub frame pages.
+- Developers can modify the default audio output for the top-level frame and all other sub frames from the same-origin sub frame.
+- Any media element or audio context can continue to override this default setting using the existing setSinkId API.
+
+## Non Goals
+- A change notification for default audio output is not in scope of this project.
+- Any permission policy requirement for calling the API is not in scope of this project.
+
+## Use Cases
+
+### 1st case
+Communication app customers may use external headsets for calls, separate from their computer’s default audio option. For instance, they might have their default setup to play music through speakers, but would always want to have (potentially private) calls through a Bluetooth headset. Today, many communication apps have a setting for customers to pick their preferred output device.
+
+Communication apps may also include a variety of other applications embedded inside of it. These embedded applications may play audio during a meeting in addition to the normal audio of the call. For example, like when a video is embedded in a slide show presentation application.
+
+### 2nd case
+A User:
+A user actively engages in online meetings using a browser communication app. During these meetings, the presenter embeds live PowerPoint presentations within iframes. These embedded presentations often include crucial content from video share site.
+However, a user faces a delightful dilemma: her children are playing in the backyard, and she wants to play music for them using a Bluetooth speaker. To achieve this, she cleverly configures her system settings:
+
+- Bluetooth Speaker: A user sets the system default audio output to the Bluetooth speaker, ensuring her children enjoy their music outdoors.
```

Comment:
Instead of using the OS settings, could we please use the new API we're proposing to set the default audio endpoint through the music web app's settings?